sig-scalability/2020-annual-report.md

```diff
@@ -0,0 +1,112 @@
+# 2020 - SIG Scalability - Community Meeting Annual Report
+
+## Operational
+
+**How are you doing with operational tasks in sig-governance.md?**
+
+- **Is your README accurate? have a CONTRIBUTING.md file?**
+
+The README is accurate an up-to-date.
+We don't have CONTRIBUTING.md and it's in our short-term plans to update
+the [developer documentation].
```

Comment:
Thank you 🙏 
Just an FYI - If you don't have the bandwidth to update the developer docs, [calling out the things things that are out-dated and/or should be added](https://github.com/kubernetes/community/issues/5236) would be a great start.